service/src/main/java/bio/terra/workspace/common/logging/model/ActivityFlight.java

```diff
@@ -0,0 +1,94 @@
+package bio.terra.workspace.common.logging.model;
+
+import bio.terra.workspace.common.exception.UnknownFlightClassNameException;
+import bio.terra.workspace.service.resource.controlled.cloud.gcp.ainotebook.UpdateControlledAiNotebookResourceFlight;
+import bio.terra.workspace.service.resource.controlled.cloud.gcp.bqdataset.UpdateControlledBigQueryDatasetResourceFlight;
+import bio.terra.workspace.service.resource.controlled.cloud.gcp.gcsbucket.UpdateControlledGcsBucketResourceFlight;
+import bio.terra.workspace.service.resource.controlled.flight.clone.bucket.CloneControlledGcsBucketResourceFlight;
+import bio.terra.workspace.service.resource.controlled.flight.clone.dataset.CloneControlledGcpBigQueryDatasetResourceFlight;
+import bio.terra.workspace.service.resource.controlled.flight.clone.workspace.CloneAllResourcesFlight;
+import bio.terra.workspace.service.resource.controlled.flight.clone.workspace.CloneGcpWorkspaceFlight;
+import bio.terra.workspace.service.resource.controlled.flight.create.CreateControlledResourceFlight;
+import bio.terra.workspace.service.resource.controlled.flight.delete.DeleteControlledResourceFlight;
+import bio.terra.workspace.service.resource.referenced.flight.create.CreateReferenceResourceFlight;
+import bio.terra.workspace.service.resource.referenced.flight.update.UpdateReferenceResourceFlight;
+import bio.terra.workspace.service.workspace.flight.CreateGcpContextFlightV2;
+import bio.terra.workspace.service.workspace.flight.DeleteAzureContextFlight;
+import bio.terra.workspace.service.workspace.flight.DeleteGcpContextFlight;
+import bio.terra.workspace.service.workspace.flight.RemoveUserFromWorkspaceFlight;
+import bio.terra.workspace.service.workspace.flight.WorkspaceCreateFlight;
+import bio.terra.workspace.service.workspace.flight.WorkspaceDeleteFlight;
+import bio.terra.workspace.service.workspace.flight.application.able.ApplicationAbleFlight;
+import bio.terra.workspace.service.workspace.flight.create.azure.CreateAzureContextFlight;
+import java.util.Arrays;
+
+/**
+ * All the workspace manager flights, contains the flight class name and the change target of the
```

Comment:
Can you also update DEVELOPMENT.md to reflect that developers should update this enum when adding a new flight?